### PR TITLE
Add sidekiq

### PR DESCRIPTION
**Pull request by dspencer001**

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```
Comment (bfcoder) [inline bin/setup:1]: delete this file. Don't need it.

File: lib/scry/sidekiq/sidekiq.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+:queues:
```
Comment (bfcoder) [inline lib/scry/sidekiq/sidekiq.yml:1]: probably should add `:concurrency: 10` so it limits the workers?

File: scry.gemspec [hunk 3]
```diff
@@ -21,6 +22,8 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "bundler"
   gem.add_development_dependency "pry-byebug"
   gem.add_development_dependency "rspec"
+  gem.add_dependency "sidekiq", "~> 4.2"
```
Comment (bfcoder) [inline scry.gemspec:25]: I don't think `add_dependency` is a thing. I think this should be down in the array on line 30 with the `add_runtime_dependency` call

File: scry.gemspec [hunk 4]
```diff
@@ -21,6 +22,8 @@ Gem::Specification.new do |gem|
...
+  gem.add_dependency "thin", "~> 1.7"
```
Comment (bfcoder) [inline scry.gemspec:26]: Same here

File: lib/scry/sidekiq/config.ru [hunk 5]
```diff
@@ -0,0 +1,8 @@
+require 'sidekiq'
+
+Sidekiq.configure_client do |config|
+  config.redis = { :size => 1 }
```
Comment (bfcoder) [inline lib/scry/sidekiq/config.ru:4]: `size: 1`

File: scry.gemspec (hunk 3 above)
Comment (dspencer001) [reply scry.gemspec:25]: It's an alias for add_runtime_dependency. I'll move them to the array though.

Comment (bfcoder) [reply scry.gemspec:25]: ahh, I just couldn't find it in http://guides.rubygems.org/specification-reference/

**Pull request closed without merging**